modules/xmpp/JingleSessionPC.js

```diff
@@ -56,9 +57,8 @@ function JingleSessionPC(me, sid, peerjid, connection,
     // stable and the ice connection state is connected.
     this.modifySourcesQueue.pause();
 }
-//XXX this is badly broken...
-JingleSessionPC.prototype = JingleSession.prototype;
-JingleSessionPC.prototype.constructor = JingleSessionPC;
+
+util.inherits(JingleSessionPC, JingleSession);
```

Comment:
@lyubomir, you have already added babel to lib-jitsi-meet. So maybe we should use ES6 classes and extends instead?